java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -313,8 +313,11 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_hash(JNIEnv *env, jobje
   try {
     auto column_views =
         cudf::jni::native_jpointerArray<cudf::column_view>{env, column_handles}.get_dereferenced();
-    return release_as_jlong(cudf::hash(cudf::table_view{column_views},
-                                       static_cast<cudf::hash_id>(hash_function_id), seed));
+    if (hash_function_id == 2) { // MD5 from HashType.java
```

Comment:
Looking for some advice on handling this since the `cudf::hash_id` has been repurposed for partitioning-hash only now and so does not include MD5.

Reply:
I think we can remove support for MD5 from this method (and from HashType.java).  The ColumnVector.md5Hash method can call a new, dedicated private native method for MD5 hashing rather than relying on passing an enum to the private native `hash` method.